content/docs/deployment/csminstallationwizard/src/static/js/commands.js

```diff
@@ -16,10 +16,10 @@
  *
  */
 var commandTitle = 'Run the following commands to install';
-var commandNote = 'Ensure that the namespaces and secrets are created before installing the helm chart';
+var commandNote = 'Ensure that the namespaces and secrets are created before installing the driver';
 var command1 = 'helm repo add dell https://dell.github.io/helm-charts';
 var command2 = 'helm install $release-name dell/container-storage-modules -n $namespace --version $version -f values.yaml';
-
+var command3 = 'kubectl create -f values.yaml';
```

Comment:
Can this be changed from values.yaml to storage_csm_$array_v$version.yaml and replaced with right values?